libs/core/src/lib/breadcrumb/breadcrumb.component.html

```diff
@@ -61,8 +61,8 @@
                 fd-link
                 tabindex="0"
                 class="fd-breadcrumb__collapsed"
-                (keydown.enter)="_keyDownHandle($event)"
-                (keydown.space)="_keyDownHandle($event)"
+                (keydown.enter)="_overflowKeyDownHandle($event)"
```

Comment:
Overflow layout supports keyboard navigation via `enableKeyboardNavigation` input property and adding `fdOverflowLayoutFocusableItem` directive to the item that can be focused.

Reply:
This line you've highlighted here is for the overflow menu button. The actual breadcrumb links themselves are coming from the app developer through `ng-content`.  Correct me if I am wrong but I think they would need to provide this directive as well through the `fd-link`s, and import the overflow module, so I think what I have in this PR is simpler for the developer

Reply:
You can provide focusableItem as fd-link. Moreover, you can extend focusableItem directive to search for the focusableItem element inside directives content, which would be much better on my opinion